app/src/app/components/specmath-modal/modal.component.ts

```diff
@@ -15,50 +15,58 @@
 import { Component, OnInit } from '@angular/core';
 import { MatDialogRef } from '@angular/material/dialog';
 import { MatStepper } from '@angular/material/stepper';
-import { SpecNameInputOptions, DefaultsFileUploadOptions } from 'src/shared/interfaces';
+import { SpecNameInputOptions, DefaultsFileUploadOptions, SpecFilesUploadOptions } from 'src/shared/interfaces';
 
-type FileUpload = 'default' | 'spec';
+const toolTipText = {
+  1: 'You must name your new spec',
+  3: 'You must upload a set of spec files'
+};
 
 @Component({
   selector: 'app-modal',
   templateUrl: './modal.component.html',
   styleUrls: ['./modal.component.scss']
 })
 export class ModalComponent implements OnInit {
-  minSteps: number;
-  currentStep: number;
-  maxSteps: number;
-  specNameInputOptions?: SpecNameInputOptions;
-  defaultsFileUploadOptions?: DefaultsFileUploadOptions;
+  MAX_STEPS = 4;
+  MIN_STEPS = 1;
+  currentStep = this.MIN_STEPS;
```

Comment:
Could we use an enum for the current step instead of the magic numbers? We might have to change up some of the math we use right now, but should make this code clearer overall.

Reply:
Updated